packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -90,4 +123,67 @@ describe("<DatePicker />", () => {
 
     expect(button).toHaveFocus()
   })
+
+  describe("Validation", () => {
+    it("displays error message when status is error", async () => {
+      render(
+        <DatePicker
+          {...defaultProps}
+          status="error"
+          validationMessage="Invalid Date."
+        />
+      )
+
+      expect(screen.getByText("Invalid Date.")).toBeInTheDocument()
+    })
+
+    it("displays error message when selected day is invalid", async () => {
+      render(<MockDatePicker selectedDay={new Date("potato")} />)
+
+      expect(screen.getByText("Date is invalid")).toBeInTheDocument()
+    })
+
+    it("displays error message when selected day is disabled", async () => {
+      render(
+        <MockDatePicker
+          disabledBefore={new Date()}
```

Comment:
```suggestion
          disabledBefore={new Date(2022, 4, 15)}
```
Best to be explicit with set values.